models/classes/user/OntologyLtiUserService.php

```diff
@@ -68,15 +68,15 @@ protected function updateUser(LtiUserInterface $user, LtiLaunchData $ltiContext)
             ]);
 
             foreach ($properties as $key => $values){
-                if ($values != $user->getPropertyValues($key)){
+                if ($values !== $user->getPropertyValues($key)){
```

Comment:
Strict comparison will trigger update 5 times. $values is object[ ].
Not strict comparison will trigger update 1 time even if nothing changed. In last comparison $values is Resource[]. 
```
 public function __toString()
    {
        return $this->getUri()."\n" . $this->getLabel() ;
    }
```
This is not equal to just Uri from right side in any case.